apps/github-monday-code/SETUP.md

```diff
@@ -24,8 +24,6 @@ To integrate with Github, you need to create an OAuth app using your Github acco
 
 1. Copy the url outputted by running the `npm run dev` command. For example: `https://happy-pig-99.tunnel.monday.app`.
```

Comment:
@yuhgto also we should change the example here

Reply:
done